### ISSUE TITLE
export

### ISSUE THREAD

**Issue by harshsbhat** [harshsbhat/sealnotes#12]
Exporting notes should be allowed!

**Comment by SuperALKALINEdroiD** [comment]
Hello, I can try this. I was thinking exporting content to a password protected file. What do you say? Lets discuss

**Comment by harshsbhat** [comment]
How would you do that?

**Comment by SuperALKALINEdroiD** [comment]
there are libraries to do this or we could brainstorm something

**Comment by harshsbhat** [comment]
Nah, I would like to hear the entire process of how you would do it. This makes sure that you don't end up doing anything that is not planned.

**Comment by SuperALKALINEdroiD** [comment]
I will draft a plan tomorrow

Sarthak Sharma
sarthaksh.netlify.app

Sent from Proton Mail Android

-------- Original Message --------

**Comment by harshsbhat** [comment]
Sure no rush
